fastlane/Fastfile

```diff
@@ -541,6 +541,58 @@ lane :beta do
   )
 end
 
+lane :download_app_store_reviews do
+  desc "Download App Store reviews and save them to a file"
+
+  UI.message "Logging into App Store Connect..."
+
+  # Use credentials from the Appfile
+  apple_id = CredentialsManager::AppfileConfig.try_fetch_value( :apple_id )
+  itc_team_id = CredentialsManager::AppfileConfig.try_fetch_value( :itc_team_id )
+
+  # Explicitly login with credentials
```

Comment:
I think this could use some error handling. What if login fails?

Reply:
I think that is fine as is, if the login fails the script also just fails with an error message.